raven/src/main/java/com/getsentry/raven/connection/BufferedConnection.java

```diff
@@ -158,6 +194,14 @@ public void run() {
                 while (events.hasNext() && !closed) {
                     Event event = events.next();
 
+                    long now = System.currentTimeMillis();
+                    long eventTime = event.getTimestamp().getTime();
+                    long age = now - eventTime;
+                    if (age < minAgeMillis) {
+                        logger.trace("Ignoring buffered event because it only " + age + "ms old.");
+                        return;
+                    }
```

Comment:
So now that we store events before being sent, there is a race where you write to disk and the `Flusher` may happen to run and immediately attempt to send it while the "main" connection is also sending.

I figured it was,

1. Not 100% critical to never, ever send dupes, and not deserving of a much more elaborate system to rule out "who is sending what event this second."
2. Enough to only attempt to send buffered send events once they are at least `flushTime` (default 60s) old.